packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -1448,20 +1448,42 @@ const consistentPatchProperties = (patchOp, _opts, ctx) => {
 
 const SYNC_DELETE_RESPONSES = ["200", "204", "default"];
 const LR_DELETE_RESPONSES = ["202", "204", "default"];
-const SYNC_ERROR = "Synchronous delete operations must have responses with 200, 204, and default return codes. They also must have no other response codes.";
-const LR_ERROR = "Long-running (LRO) delete operations must have responses with 202, 204, and default return codes. They also must have no other response codes.";
+const SYNC_ERROR = "Synchronous delete operations must have responses with 200, 204 and default return codes. They also must have no other response codes.";
+const LR_ERROR = "Long-running (LRO) delete operations must have responses with 202, 204 and default return codes. They also must have no other response codes.";
+const EmptyResponse_ERROR = "Delete operation response codes MUST be non-empty. It MUST have response codes 200, 204 and default if it is sync OR 202, 204 and default if it is long running.";
```

Comment:
Nit - About "MUST" and "OR", I see you uppercased to help the user notice the issue. I would suggest lowercase.

For example - "The response codes for a delete operation must not be empty. The response code should be either 200 or 204, if the operation is synchronous. If the operation is async and long running, then the response code should be either 202 or 204."

Reply:
modified casing